xdsl/dialects/vector.py

```diff
@@ -169,4 +169,76 @@ def get(memref: SSAValue | Operation, indices: List[SSAValue | Operation],
                                 ])
 
 
-Vector = Dialect([Load, Store, Broadcast, FMA, Maskedload], [])
+@irdl_op_definition
+class Maskedstore(Operation):
+    name = "vector.maskedstore"
+    memref: Annotated[Operand, MemRefType]
+    indices: Annotated[VarOperand, IndexType]
+    mask: Annotated[Operand, VectorType]
+    value_to_store: Annotated[Operand, VectorType]
+
+    def verify_(self):
+        memref_element_type = self.memref.typ.element_type
+
+        if memref_element_type != self.value_to_store.typ.element_type:
+            raise VerifyException(
+                "MemRef element type should match the stored vector type. Obtained types were "
+                + str(memref_element_type) + " and " +
+                str(self.value_to_store.typ.element_type) + ".")
+
+        if len(self.value_to_store.typ.get_shape()) != 1:
+            raise VerifyException("Expected a rank 1 vector to be stored.")
+
+        if len(self.mask.typ.get_shape()) != 1:
+            raise VerifyException("Expected a rank 1 mask vector.")
+
+        if self.mask.typ.element_type != i1:
+            raise VerifyException("Expected mask element type to be i1.")
+
+        if self.memref.typ.get_num_dims() != len(self.indices):
+            raise VerifyException(
+                "Expected an index for each memref dimension.")
+
+    @staticmethod
+    def get(memref: SSAValue | Operation, indices: List[SSAValue | Operation],
+            mask: SSAValue | Operation,
+            value_to_store: SSAValue | Operation) -> Maskedstore:
+        return Maskedstore.build(
+            operands=[memref, indices, mask, value_to_store])
+
+
+@irdl_op_definition
+class Print(Operation):
+    name = "vector.print"
+    source: Annotated[Operand, Attribute]
```

Comment:
Please use `AnyAttr()` here. The `Attribute` annotation is correct, but feels a bit weird, IMO. AnyAttr nicely shows that this is just aynthing.